Custom Field Constraints/CFConstraints.data.provider.js

```diff
@@ -98,6 +100,38 @@ tau.mashups
                 return getTasksDeferred.promise();
             },
 
+            getTeamProjectsPromise: function(entitiesDetailed) {
+                var getTeamProjectsDetails = $.Deferred();
+                var teamProjectIds = _.chain(entitiesDetailed)
+                    .filter(function(entityDetailed) {
+                        return entityDetailed.project &&
+                            entityDetailed.assignedTeams && entityDetailed.assignedTeams.length > 0;

```

Comment:
`!_.isEmpty(entityDetailed.assignedTeams)` is less verbose then `entityDetailed.assignedTeams && entityDetailed.assignedTeams.length > 0`